frontend/src/component/archive/ArchiveTable/ArchivedFeatureActionCell/ArchivedFeatureReviveConfirm/ArchivedFeatureReviveConfirm.tsx

```diff
@@ -0,0 +1,98 @@
+import { Alert, styled } from '@mui/material';
+import React from 'react';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { formatUnknownError } from 'utils/formatUnknownError';
+import useToast from 'hooks/useToast';
+import useProjectApi from 'hooks/api/actions/useProjectApi/useProjectApi';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+interface IArchivedFeatureReviveConfirmProps {
+    revivedFeatures: string[];
+    projectId: string;
+    open: boolean;
+    setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    refetch: () => void;
+}
+
+const StyledParagraph = styled('p')(({ theme }) => ({
+    marginTop: theme.spacing(2),
+}));
+
+export const ArchivedFeatureReviveConfirm = ({
```

Comment:
I'd add a test that verifies that revive API was called, that refetch was invoked and that correct toast data appears. We have a pattern how to easily test those example: https://github.com/Unleash/unleash/pull/4983/files